src/ir/context.rs

```diff
@@ -1038,13 +1105,14 @@ impl<'ctx> BindgenContext<'ctx> {
                         let sub_item = Item::new(sub_id,
                                                  None,
                                                  None,
-                                                 template_decl_id,
+                                                 self.current_module,
```

Comment:
I just changed from one well-known-but-sort-of-bogus parent to another well-known-but-sort-of-bogus parent.

They used to always be parented by the template definition, which doesn't really make sense either, just happened to be handy when the "real" parent wasn't.

I can definitely add a comment.